src/EditorWithXStateImports.tsx

```diff
@@ -105,96 +104,85 @@ export const EditorWithXStateImports = (
   }, [editorTheme.theme]);
 
   return (
-    <ClassNames>
-      {({ css }) => (
-        <Editor
-          wrapperClassName={`${css`
-            min-height: 0;
-            min-width: 0;
-          `} js-monaco-editor`}
-          defaultPath="main.ts"
-          defaultLanguage="typescript"
-          defaultValue={props.defaultValue}
-          options={{
-            minimap: { enabled: false },
-            tabSize: 2,
-            glyphMargin: true,
-          }}
-          loading={<SpinnerWithText text="Preparing the editor" />}
-          onChange={(text) => {
-            if (typeof text === 'string') {
-              props.onChange?.(text);
-            }
-          }}
-          theme="vs-dark"
-          onMount={async (editor, monaco) => {
-            editorRef.current = monaco.editor;
-            const theme = editorTheme.theme;
-            monaco.editor.defineTheme(theme, themes[theme]);
-            monaco.editor.setTheme(theme);
-
-            monaco.languages.typescript.typescriptDefaults.setWorkerOptions({
-              customWorkerPath: `${new URL(
-                window.location.origin,
-              )}viz/ts-worker.js`,
-            });
-
-            monaco.languages.typescript.typescriptDefaults.setCompilerOptions({
-              ...monaco.languages.typescript.typescriptDefaults.getCompilerOptions(),
-              module: monaco.languages.typescript.ModuleKind.CommonJS,
-              moduleResolution:
-                monaco.languages.typescript.ModuleResolutionKind.NodeJs,
-              strict: true,
-            });
-
-            // Prettier to format
-            // Ctrl/CMD + Enter to visualize
-            editor.addAction({
-              id: 'format',
-              label: 'Format',
-              keybindings: [monaco.KeyMod.CtrlCmd | monaco.KeyCode.Enter],
-              run: (editor) => {
-                editor.getAction('editor.action.formatDocument').run();
-              },
-            });
-
-            monaco.languages.registerDocumentFormattingEditProvider(
-              'typescript',
-              {
-                provideDocumentFormattingEdits: (model) => {
-                  try {
-                    return [
-                      {
-                        text: prettify(editor.getValue()),
-                        range: model.getFullModelRange(),
-                      },
-                    ];
-                  } catch (err) {
-                    console.error(err);
-                  } finally {
-                    props.onFormat?.();
-                  }
+    <Editor
```

Comment:
The only change here is the removal of `<ClassNames>`, right?

Reply:
Ye, gonna try to remember to mention this kind of stuff in the PR comments next time 👍